contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,95 @@
+# Validation Suite Testing in Kubernetes
+
+A Kubernetes Validation suite is a subset of e2e tests that are used to
+validate functionality of an optional capability of Kubernetes for use
+with a specific release. Validation suites can grow or shrink over time, and can also
+change as a functionality progresses from alpha, to beta, and to GA. The flexibility
+of Validation suites to change based on experimentation and feedback is a key 
+characteristic that differentiates Validation suites from 
+[Conformance Tests](../sig-architecture/conformance-tests.md) and 
+[Conformance profiles](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md). In addition, Validation suites differ from  
+Conformance profiles in that they do not require conformance certification approval
+in order to be created and utilized.  The creation of a Conformance Profile requires
+consensus and discussion across the community because there is a desire to limit
```

Comment:
"The creation of a Conformance Profile requires
consensus and discussion across the community because there is a desire to limit
the number of conformance profiles created; a large number of Conformance Profiles
would inhibit the portability of Kubernetes workloads."

This is unrelated to subject at hand @bradtopol can we please remove this?

Reply:
Hi @dims.  Sure. Will do that right now